sympy/physics/mechanics/kane.py

```diff
@@ -55,6 +57,17 @@ class KanesMethod(_Methods):
         Boolean whether the mass matrices and forcing vectors should use the
         explicit form (default) or implicit form for kinematics.
         See the notes for more details.
+    kd_eqs_solver : str, callable
+        Solver to be used for the kinematic differential equations. The default
+        is LU solve. See the notes for more details on what the solver is used
+        for and advise on what solver should be used.
```

Comment:
How does this interplay with the `explicit_kinematics` option above?

Reply:
There should not be any problems between selecting a linear solver and specifying `explicit_kinematics` differently.